templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/delete_vm_extensions.sh

```diff
@@ -27,7 +27,6 @@ terraform init -input=false -backend=true \
 echo "Running terraform state list"
 tf_state_list="$(terraform state list)"
 echo "State list result: ${tf_state_list}"
-echo "${tf_state_list}" | grep "azurerm_virtual_machine_extension." | xargs -r terraform state rm
+echo "${tf_state_list}" | { grep "azurerm_virtual_machine_extension." || [[ $? == 1 ]]; } | xargs -r terraform state rm
```

Comment:
@tanya-borisova - after the discussion this morning, I don't think that my suggestion will fix the issue, so I think you're changes are probably still needed. I think it would be worth adding a comment to explain the changes in the script to explain the change.

If we want to add output for what extensions we're removing from the state then moving to a `for` loop probably makes sense to also include.

Reply:
> If we want to add output for what extensions we're removing from the state then moving to a for loop probably makes sense to also include.

There is still output, Terraform logs what it is removing